### ISSUE TITLE
Link to unloadRecord broken

### ISSUE THREAD

**Issue by raulb** [ember-learn/ember-api-docs#629]
![]https://d.pr/free/i/dimyGJ.gif)

The reference to Ember Data is correct, but neither of those two links (see screenshot) bring you to the right place which should be:

It should link to https://api.emberjs.com/ember-data/3.12/classes/Model/methods/unloadRecord?anchor=unloadRecord

**Comment by toddjordan** [comment]
yeah, seems to be only happening on the landing page.  When you are in ember api docs you don't get the ember data results.  When you are in ember data api docs you get the results and the link works.

cc @sivakumar-kailasam
